cts/cts-cli.in

```diff
@@ -2890,30 +2890,16 @@ class ValidityRegressionTest(RegressionTest):
             Test("Try to make resulting CIB invalid (enum violation)",
                  """cibadmin -M -o constraints --xml-text '<rsc_order id="ord_1-2" first="dummy1" first-action="break" then="dummy2"/>'""",
                  expected_rc=ExitStatus.CONFIG, update_cib=True),
-            Test("Run crm_simulate with invalid CIB (enum violation)",
-                 "crm_simulate -p -S",
-                 stdin=StdinCmd("""cibadmin -Q | sed 's#"start"#"break"#'"""),
-                 expected_rc=ExitStatus.CONFIG),
             Test("Try to make resulting CIB invalid (unrecognized validate-with)",
                  "cibadmin -M --xml-text '<cib validate-with=\"pacemaker-9999.0\"/>'",
                  expected_rc=ExitStatus.CONFIG, update_cib=True),
-            Test("Run crm_simulate with invalid CIB (unrecognized validate-with)",
-                 "crm_simulate -p -S",
-                 stdin=StdinCmd("""cibadmin -Q | sed 's#"pacemaker-1.2"#"pacemaker-9999.0"#'"""),
-                 expected_rc=ExitStatus.CONFIG),
             Test("Try to make resulting CIB invalid, but possibly recoverable (valid with X.Y+1)",
                  "cibadmin -C -o configuration --xml-text '<tags/>'",
                  expected_rc=ExitStatus.CONFIG, update_cib=True),
-            Test("Run crm_simulate with invalid, but possibly recoverable CIB (valid with X.Y+1)",
-                 "crm_simulate -p -S",
-                 stdin=StdinCmd("cibadmin -Q | sed 's#</configuration>#<tags/></configuration>#'")),
             Test("Make resulting CIB valid, although without validate-with attribute",
                  "cibadmin -p -R",
                  stdin=StdinCmd("""cibadmin -Q | sed 's#[ 	][ 	]*validate-with="[^"]*"##'"""),
                  update_cib=True),
-            Test("Run crm_simulate with valid CIB, but without validate-with attribute",
```

Comment:
Why keep the final `crm_simulate` one ("Run crm_simulate with invalid CIB, also without validate-with attribute")?

Reply:
It's no longer possible to run crm_simulate on a CIB with a missing or unknown validate-with, so those had to be dropped. Keeping one crm_simulate test with validate-with="none" seemed worthwhile to ensure the nonconforming syntax is actually getting through (as long as we support that).